findimports.py

```diff
@@ -962,10 +962,12 @@ def printUnusedImports(self):
                         continue
                 print(f"{module.filename}:{lineno}: {name} not used")
 
-    def constructDot(self):
+    def constructDot(self, attributes=()):
         """Produce a dependency graph in dot format."""
         lines = list()
         lines.append("digraph ModuleDependencies {")
+        if attributes:
+            lines.extend(map("  {}".format, attributes))
```

Comment:
Whoa, that's clever.  For some reason I assumed we'd have to split the k=v pairs when parsing arguments into a dict, and then reconstruct it here.

Reply:
Also, I still want the semicolons for consistency, even if the dot grammar makes them optional.

Reply:
Operation `ensure-semicolon-at-end` will be more complex than the whole addition.

It is not currently absent, it is just left to the user. E.g,:
```
--attr "id=id"
--attr "id=id;"
--attr "id=id;id=id"
```

Reply:
Yeah, I'm not comfortable allowing arbitrary graphviz syntax here.    If the user does `-A "title=blah; blah"`, we should do string escaping etc. and insert

    title="blah; blah";

in the generated .dot.  (Is there a title attibute for graphs?)

Reply:
Ok, can do that.
One can do: `label = "process #1";`